src/nrniv/nrncore_write.cpp

```diff
@@ -367,7 +367,7 @@ int nrncore_is_file_mode() {
     return 0;
 }
 
-int nrncore_psolve(double tstop) {
+int nrncore_psolve(double tstop, int file_mode) {
```

Comment:
I don't think this change was intended to be in this pull request?

Reply:
You are correct. However, without it, the windows version does not compile with gcc-10. It is so trivial I thought I'd just drag it along.